### PR TITLE
feature: Webpack Stats from Environment

### PR DESCRIPTION
**Pull request by icirellik**
Add support for loading the webpack datadog configuration from the
shared environment file and log some basic machine stats.

### CODE REVIEW

File: webpack/utils/env.js [hunk 1]
```diff
@@ -1,17 +1,51 @@
...
+  try {
+    accessSync(filepath, constants.F_OK)
+  } catch (e) {
+    return false
+  }
+  return true
+}
+
+// Load the default envs and apply to process.env
+applyToEnv({
```
Comment (ovasdi) [inline webpack/utils/env.js:34]: This looks like an improved `src/loadenv.js` implementation! Wondering why you didn't refactor that and `import src/loadenv.js` here instead?

**Review by ovasdi — APPROVED** [review]
LGTM ❗

File: webpack/utils/env.js (hunk 1 above)
Comment (icirellik) [reply webpack/utils/env.js:34]: I debated heavily, this actually started as a refactor of `src/loadenv.js`. My preference is to keep our build source completely separate from our application logic lest it accidentally be included in a production build by webpack due to a rogue import.

It made me think twice about making a separate npm module for it 😄

Comment (ovasdi) [reply webpack/utils/env.js:34]: Thanks for clarifying. Let the telemetry roll in 😃

**Pull request merged**